test/scanner/secretInLog.spec.ts

```diff
@@ -23,3 +23,20 @@ it('secret in log file', async () => {
     expect(finding.event.id).toEqual(817);
   }
 });
+
+it('parses out multiple secrets from function return value', async () => {
+  const check = new Check(rule);
+  const findings = await scan(
+    check,
+    'appmaps/secretInLog/Confirmation_already_confirmed_user_should_not_be_able_to_confirm_the_account_again.appmap.json'
```

Comment:
This is an AppMap from Devise that tests the case when multiple secrets are emitted from a secret-labeled function.